index.html

```diff
@@ -1,47 +1,152 @@
 <body>
-    <header>
-        <nav class="toolbar" id="toolbar">
-            <a hre="#" class="logo">My Logo</a>
-            <div class="menu">
-                <button class="dropdown-button"><span class="material-symbols-outlined">
-                    menu
-                    </span></button>
-                <ul class="dropdown-menu">
-                    <a href="#">Portfolio</a>
-                    <a href="#">About</a>
-                    <a href="#">Contact</a>
-                    <a href="#"><span class="material-symbols-outlined">
-                        mail
-                        </span></a>
-                </ul>
-            </div>
-        </nav>
-    </header>
-    <section class="headline">
-        
-        <h1>Hey There.<br>I'm Tomás</h1>
-        <h2>I'm a web developer</h2>
-        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
-        <ul class="social-media">
-            <li><a href="#"><img src="./Normal_Button/Icon-GitHub.png"></a></li>
-            <li><a href="#"><img src="./Normal_Button/Icon-Linkedin.png"></a></li>
-            <li><a href="#"><img src="./Normal_Button/angelist-1.png"></a></li>
-            <li><a href="#"><img src="./Normal_Button/Icon-Twitter.png"></a></li>
-            <li><a href="#"><img src="./Normal_Button/Icon-Medium.png"></a></li>
-        </ul>
-        </div>
-    </section>
+  <header>
+    <nav class="toolbar" id="toolbar">
+      <a hre="#" class="logo">My Logo</a>
+      <div class="menu">
+        <button class="dropdown-button"><span class="material-symbols-outlined">menu</span>
+        </button>
+        <ul class="dropdown-menu">
+          <a href="#work-section">Portfolio</a>
+          <a href="#about-section">About</a>
+          <a href="#">Contact</a>
+          <a href="#"><span class="material-symbols-outlined">mail</span></a>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <section class="headline">    
+    <h1>Hey There.<br>I'm Tomás</h1>
+    <h2>I'm a web developer</h2>
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+    <ul class="social-media">
+      <li><a href="#"><img src="./Normal_Button/Icon-GitHub.png"></a></li>
+      <li><a href="#"><img src="./Normal_Button/Icon-Linkedin.png"></a></li>
+      <li><a href="#"><img src="./Normal_Button/angelist-1.png"></a></li>
+      <li><a href="#"><img src="./Normal_Button/Icon-Twitter.png"></a></li>
+      <li><a href="#"><img src="./Normal_Button/Icon-Medium.png"></a></li>
+    </ul>
+  </section>
+  <section class="work-section" id="work-section">
+    <div class="work">
+      <h1 class="grid1">My recent works<img src="./Normal_Button/Vector.png"></h1> 
+      <div class="grid2">
```

Comment:
As specified by the previous reviewer. Please use only one h1 tag in each of your documents. This is for best SEO practice. Kindly change this h1 tag to any other heading tag between `h2` and `h5`